### PR TITLE
Updates Folder Structure to Match CSS Guidelines

### PR DESCRIPTION
**Pull request by janiv**
Updates the folder structure of the src/scss folder to match the
suggestions in the guidenlines.

Adds .inline-header, .emphasis, and margin-bottom--none classes.

### CODE REVIEW

**Comment by janiv** [comment]
@jehoshua02 Will you look at this?

File: dist/scss/base/_margins.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.margin-bottom--none {
+  margin-bottom: 0 !important; // When you ask for margin-bottom none, you better get it.
```
Comment (jehoshua02) [inline dist/scss/base/_margins.scss]: @janiv This belongs in `trumps/` or no?

File: dist/scss/base/_typography.scss [hunk 2]
```diff
@@ -17,3 +17,7 @@ body {
     line-height: 1.2;
     color: $darker-gray;
 }
+
+.emphasis {
```
Comment (jehoshua02) [inline dist/scss/base/_typography.scss]: @janiv I'd expect `.emphasis` to do what `<em>` does. And `.strong` to do what `<strong>` does.

<em>emphasis</em>
<strong>strong</strong>

File: dist/scss/components/_headers.scss [hunk 3]
```diff
@@ -37,3 +37,7 @@ h5 {
 h6 {
   font-size: $font-size-h6;
 }
+
+.inline-header {
```
Comment (jehoshua02) [inline dist/scss/components/_headers.scss]: @janiv

`.inline-header` or `.header--inline`?

Also `h1` or `.header--h1`?

Comment (jehoshua02) [inline dist/scss/components/_headers.scss]: I ask because I'm still learning how to think with BEM.

File: dist/scss/generics/box-sizing.scss [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+*,
+*:before,
+*:after {
+    -webkit-box-sizing: inherit; // Safari/Chrome, other WebKit
+    -moz-box-sizing: inherit;    // Firefox, other Gecko
+    box-sizing: inherit;         // Opera/IE 8+
+}
+
+html {
```
Comment (jehoshua02) [inline dist/scss/generics/box-sizing.scss]: @janiv I wonder if we should have a set of mixins? And we can eliminate prefixes with [gulp-autoprefixer](https://www.npmjs.com/package/gulp-autoprefixer) 

```scss
@mixin box-sizing($value) {
  & {
    box-sizing: border-box;
    & *:before, & *:after {
      box-sizing: inherit;
    }
  }
}

@mixin border-box() {
  @include box-sizing(border-box);
}

@mixin content-box() {
  @include box-sizing(content-box);
}
```

```scss
html {
  @include border-box;
}

// ...

.icon {
  @include content-box;
}
```

**Comment by jehoshua02** [comment]
@janiv Sorry if some of my comments aren't strongly related to the changes made.

Run `gulp clean` then `gulp build` to prune leftover files.

**Comment by janiv** [comment]
This is all great feedback. Will make these changes. Thank you. 

—

**Comment by jehoshua02** [comment]
@janiv You are welcome. Great work.

**Comment by janiv** [comment]
@jehoshua02 I've made all changes requested. Thoughts on the last commit? Are we good to merge and tag?

**Comment by jehoshua02** [comment]
@janiv Other than that one selector, looks good!

**Pull request merged**